### PR TITLE
List Sub Text (4 lines): add example and update style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/1054
[Problem] Missing example and out of dated styles
[Solution] Add example and update style
    
Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+						List 01
+						<span class="ui-li-sub-text" >
+							Sub text
+						</span>
+						<span class="ui-li-sub-text" >
+							12:45AM
+						</span>
+					</a>
+				</li>
+				<li class="li-has-multiline">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:37]: add "li-has-2line-sub" class.

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+						List 02
+						<span class="ui-li-sub-text">
+							Sub text
+						</span>
+						<span class="ui-li-sub-text" >
+							12:45AM
+						</span>
+					</a>
+				</li>
+				<li class="li-has-multiline">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:48]: add "li-has-2line-sub" class.

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+			<h2 class="ui-title">
+				Title
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li class="li-has-multiline li-has-2line-sub">
+					<a>
+						List 01
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:29]: last 1 space remove.

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+							Sub text
+						</span>
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:32]: last 1 space remove.

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+						</span>
+					</a>
+				</li>
+				<li class="li-has-multiline">
+					<a>
+						List 02
+						<span class="ui-li-sub-text">
+							Sub text
+						</span>
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:43]: last 1 space remove.

File: examples/wearable/UIComponents/contents/presentation-views/list/3-line.html [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+						</span>
+					</a>
+				</li>
+				<li class="li-has-multiline">
+					<a>
+						List 03
+						<span class="ui-li-sub-text">
+							Sub text
+						</span>
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/3-line.html:54]: last 1 space remove.

File: examples/wearable/UIComponents/contents/presentation-views/list/4-line.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+			<h2 class="ui-title">
+				Title
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li class="li-has-multiline li-has-3line-sub">
+					<a>
+						List 01
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/4-line.html:29]: Remove the last space from the span tag in this file.

File: examples/wearable/UIComponents/contents/presentation-views/list/5-line.html [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+			<h2 class="ui-title">
+				Title
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li class="li-has-multiline li-has-4line-sub">
+					<a>
+						List 01
+						<span class="ui-li-sub-text" >
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/presentation-views/list/5-line.html:29]: Remove the last space from the span tag in this file.

File: src/css/profile/wearable/changeable/theme-circle/arclistview.less [hunk 9]
```diff
@@ -64,7 +64,7 @@
 		display: flex;
 		align-items: center;
 
-		&:not(.ui-li-group-index):not(.ui-listview-divider) {
+		&:not(.ui-li-group-index):not(.ui-listview-divider):not(.li-has-multiline) {
 			min-height: 108 * @unit_base;
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/arclistview.less:68]: not using @px_base?

File: src/css/profile/wearable/changeable/theme-circle/listview.less [hunk 10]
```diff
@@ -437,6 +450,7 @@
 						text-overflow: ellipsis;
 						display: inline-block;
 						overflow: hidden;
+
```
Comment (DohyungLim) [inline src/css/profile/wearable/changeable/theme-circle/listview.less:453]: I think no necessary line.

**Review by DohyungLim — COMMENTED** [review]
10 comment.

**Comment by grzegorz-czajkowski** [comment]
I am closing this PR due to not trivial issue related with Arclistview reported here https://github.com/Samsung/TAU/issues/1054
@DohyungLim I will apply your comments if core issue is solved.

**Pull request closed without merging**